### PR TITLE
[HYD-190] Add docs for mysql_fdw & http

### PR DESCRIPTION
**Pull request by owenthereal**
Add `mysql_fdw` & `http` to "External Table" doc

### CODE REVIEW

File: features/external_tables.md [hunk 1]
```diff
@@ -56,3 +57,90 @@ On many Postgres providers, you can also set up this connection in the other dir
...
+
+```sql
+CREATE EXTENSION mysql_fdw;
+
+CREATE SERVER remote_mysql_server FOREIGN DATA WRAPPER mysql_fdw OPTIONS (host '...', port '...');
+
+CREATE USER MAPPING FOR CURRENT_USER SERVER remote_mysql_server OPTIONS (username '...', password '...');
+```
+
+Same as `postgres_fdw`, you can now create foreign tables from your remote MySQL database.
```
Comment (wuputah) [inline features/external_tables.md]:
```suggestion
You can now create foreign tables from your remote MySQL database.
```

Avoid cross-references or assuming people have read other sections of the docs (people interested in MySQL ET are likely to have not read about Postgres ET).

File: features/external_tables.md [hunk 2]
```diff
@@ -56,3 +57,90 @@ On many Postgres providers, you can also set up this connection in the other dir
...
+Assuming your remote MySQL database's name is `mysql`, you can [import all of your remote tables](https://www.postgresql.org/docs/current/sql-importforeignschema.html) at once as follows:
+
+```sql
+CREATE SCHEMA remote_mysql;
+
+IMPORT FOREIGN SCHEMA mysql FROM SERVER remote_mysql_server INTO remote_mysql;
+```
+
+## HTTP External Sources
```
Comment (wuputah) [inline features/external_tables.md]: While HTTP is different than other ETs, do we want to introduce a new term? You can create a very similar effect to External Tables using a view, but it is read-only... so, I might suggest an example where you create a view.

**Review by wuputah — CHANGES_REQUESTED** [review]

File: features/external_tables.md (hunk 2 above)
Comment (owenthereal) [reply features/external_tables.md]: Added an example that use a view with the `http` ext: https://github.com/HydrasDB/docs/pull/7/commits/ec51b3e24385083162dcfd0f0025cc8435866c8f

File: features/external_tables.md [hunk 3]
```diff
@@ -56,3 +57,101 @@ On many Postgres providers, you can also set up this connection in the other dir
...
+  FROM http_get('https://api.github.com/orgs/hydrasdb/repos');
+```
+
+You can query the view as if it's a table. Note that Postgres view is read-only.
+
+The following gets all the repo names from the view:
+
+
+```sql
+SELECT repo_json->'name' FROM github_repos;
```
Comment (wuputah) [inline features/external_tables.md]: hmm, would it be possible to "expand" the json in the view query, to make it more table-like? Maybe you would have to do something like this:

```sql
CREATE VIEW github_repos AS
WITH repos AS (
SELECT json_array_elements(content::json) AS repo
  FROM http_get('https://api.github.com/orgs/hydrasdb/repos')
)
SELECT 
  repo->'name' AS name,
  ... -- add other fields
FROM repos;
```

**Review by wuputah — APPROVED** [review]

File: features/external_tables.md (hunk 3 above)
Comment (owenthereal) [reply features/external_tables.md]: Good call. Addressed in https://github.com/HydrasDB/docs/pull/7/commits/c4104787558171fcaaaa2d6d052be60eb5a7a19d

**Pull request merged**